addons/website_forum/views/website_forum.xml

```diff
@@ -651,13 +674,22 @@
                         </div>
                         <a t-att-href="url_for('/forum/') + slug(forum) + '?' + keep_query( 'search', 'sorting', 'filters', 'my')" class="btn py-1">&#215;</a>
                     </span>
+                    <span t-if="author" t-attf-class="w-100 w-md-auto mb-2 mb-md-0 border rounded pl-2 d-inline-flex align-items-center justify-content-between #{search and 'ml-md-2'}">
+                        <div>
+                            <img class="o_forum_avatar rounded-circle mr-1" t-att-src="website.image_url(author, 'avatar_128', '16x16')" alt="Avatar"/>
+                            <span> <b>Posts</b> by <b><em t-esc="author.name"/></b></span>
+                        </div>
+                        <a t-att-href="forum_base_url + '?' + keep_query( 'search', 'filters', 'sorting')" class="btn py-1">&#215;</a>
+                    </span>
                 </div>
-                <div t-if="uid and request.env.user.forum_waiting_posts_count"
-                    title="You already have a pending post"
-                    data-toggle="popover" data-trigger="hover" data-content="Please wait for a moderator to validate your previous post before continuing.">
-                    <a class="disabled btn btn-secondary btn-block mb-3 mb-md-0" t-attf-href="/forum/#{slug(forum)}/ask">New Post</a>
-                </div>
-                <a t-else="" role="button" type="button" class="btn btn-primary btn-block o_forum_ask_btn mb-3 mb-md-0" t-att-href="uid and '/forum/' + slug(forum) + '/ask' or '/web/login'">New Post</a>
+                <t t-if="forum">
+                    <div t-if="uid and request.env.user.forum_waiting_posts_count"
+                        title="You already have a pending post"
+                        data-toggle="popover" data-trigger="hover" data-content="Please wait for a moderator to validate your previous post before continuing.">
+                        <a class="disabled btn btn-secondary btn-block mb-3 mb-md-0" t-attf-href="/forum/#{slug(forum)}/ask">New Post</a>
+                    </div>
+                    <a t-else="" role="button" type="button" class="btn btn-primary btn-block o_forum_ask_btn mb-3 mb-md-0" t-att-href="uid and '/forum/' + slug(forum) + '/ask' or '/web/login'">New Post</a>
```

Comment:
can you use the `a if b else c` ? (to check if you use the syntax `a and b or c` elsewhere in your commit)